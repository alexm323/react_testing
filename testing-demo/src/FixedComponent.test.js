import React from 'react';
import { render, screen } from '@testing-library/react';
import FixedComponent from './FixedComponent';


test('Render a brokenComponent without crashing', () => {
    render(<FixedComponent />)
})
// snapshot test we can use test or it but they do the same thing 

test("it matches snapshot", () => {
    const { asFragment } = render(<FixedComponent />);
    expect(asFragment()).toMatchSnapshot()
})


