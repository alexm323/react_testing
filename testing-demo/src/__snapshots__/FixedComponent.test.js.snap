// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Hello, I'm the Broken Component!
    </p>
    <p>
      Here are some numbers:
    </p>
    <p>
      42
    </p>
    <p>
      43
    </p>
    <p>
      45
    </p>
  </div>
</DocumentFragment>
`;
